package.json

```diff
@@ -39,7 +39,8 @@
     "pactum": "3.2.2",
     "reflect-metadata": "0.1.13",
     "rimraf": "3.0.2",
-    "rxjs": "7.5.6"
+    "rxjs": "7.5.6",
+    "slug": "8.0.0"
```

Comment:
there is [toKebabCase](https://github.com/fullstacksjs/toolbox#tokebabcase) method in toolbox.

Reply:
slug package is not converting space to hyphen (-) only,
it converts Persian to english also.